addon/gql/queries/form-editor-question.graphql

```diff
@@ -0,0 +1,7 @@
+#import "ember-caluma-form-builder/gql/fragments/question-info"
+
+query FormEditorGeneral($id: ID!) {
```

Comment:
a bit confused on this naming. Shouldn't this be named something like `FormEditorQuestion`?

Reply:
Yes, indeed. Will change that.

Reply:
Done!